Java/Java.sublime-syntax

```diff
@@ -598,20 +573,56 @@ contexts:
 
   parens:
     - match: \(
+      scope: punctuation.section.parens.begin.java
       push:
+        - meta_scope: meta.parens.java
```

Comment:
Afaik the recommendation is to use `punctuation.section.group`, according to the scope naming guidelines. (See also punctuation scopes.)

Reply:
I don't feel strongly about this. There are so many syntaxes in the wild that use `parens` that we need to recommend they are handled by any decent color scheme. Some syntaxes parents are used where it definitely is not a "group". Similarly `brackets`.

I am leaving it up to individual syntax authors for the most part since I don't feel strongly and I'd rather spend my time on more meaningful things. 😄